src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Entities/ValidationConstants.cs

```diff
@@ -3,6 +3,8 @@
     /// <summary>
     /// Database validation constraints
     /// </summary>
+
+#pragma warning disable S1144 // False positive: https://github.com/SonarSource/sonar-csharp/issues/904
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)